go.sum

```diff
@@ -5,7 +5,9 @@ github.com/Azure/go-autorest v11.1.2+incompatible/go.mod h1:r+4oMnoxhatjLLJ6zxSW
 github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46/go.mod h1:3wb06e3pkSAbeQ52E9H9iFoQsEEwGN64994WTCIhntQ=
 github.com/PuerkitoBio/purell v1.0.0/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbtSwDGJws/X0=
 github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
+github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 h1:DDGfHa7BWjL4YnC6+E63dPcxHo2sUxDIu8g3QgEJdRY=
```

Comment:
license?

Reply:
This one is stated as MIT licensed but there is no license file in the project. How do we handle that?

Reply:
Ping @jamesbroadhead

Reply:
The project says that it's abandoned -- honestly, the best approach is probably not to use it. 

Failing that, include a copy of the MIT license
https://opensource.org/licenses/MIT

Reply:
It's a transitive dependency I'm afraid. Are we keeping track of the licenses of those?